src/Microsoft.ML.Data/Transforms/ValueMappingTransformer.cs

```diff
@@ -32,14 +32,63 @@
 
 namespace Microsoft.ML.Transforms.Conversions
 {
+    /// <summary>
+    /// The ValueMappingEstimator is a 1-1 mapping from a key to value. This particular class load the mappings from an <see cref="IDataView"/>.
+    /// This gives user the flexibility to load the mapping from file instead of using IEnumerable in <see cref="ValueMappingEstimator{TKey, TValue}"/>
+    /// </summary>
+    public class ValueMappingEstimator : TrivialEstimator<ValueMappingTransformer>
```

Comment:
rename the file to ValueMapping if both the estimator and the transformer are on the same file.

Reply:
Did this get resolved? The file is still ValueMappingTransformer.

---
In reply to: [251596169](https://github.com/dotnet/machinelearning/pull/2232#discussion_r251596169) [](ancestors = 251596169)

Reply:
Resolved if you see the sixth commit.

---
In reply to: [251634144](https://github.com/dotnet/machinelearning/pull/2232#discussion_r251634144) [](ancestors = 251634144,251596169)